editoast/src/views/infra/auto_fixes/speed_section.rs

```diff
@@ -0,0 +1,16 @@
+use super::{new_ref_fix_delete_pair, Fix};
+use crate::schema::{InfraError, InfraErrorType, ObjectRef, SpeedSection};
+use std::collections::HashMap;
+
+pub fn fix_speed_section(
+    speed_section: &SpeedSection,
+    errors: impl Iterator<Item = InfraError>,
+) -> HashMap<ObjectRef, Fix> {
+    errors
+        .filter_map(|infra_error| match infra_error.get_sub_type() {
+            InfraErrorType::EmptyObject => Some(new_ref_fix_delete_pair(speed_section)),
+            _ => None,
+        })
+        // Need to invert the ordering because removing from the front would invalidate other indexes
```

Comment:
It looks like this comment is [a teaser from the next PR](https://github.com/osrd-project/osrd/pull/6012/files#diff-1d2b51fc570cf0359344201723f5aacb780d2e11ea67e3106abd54348f20101eR53) :wink:

Reply:
Done.